library/spdm_requester_lib/libspdm_req_get_csr.c

```diff
@@ -52,6 +59,24 @@ static libspdm_return_t libspdm_try_get_csr(libspdm_context_t *spdm_context,
         return LIBSPDM_STATUS_UNSUPPORTED_CAP;
     }
 
+    if (libspdm_get_connection_version(spdm_context) == SPDM_MESSAGE_VERSION_12) {
+        if ((key_pair_id != 0) || (request_attribute != 0)) {
+            return LIBSPDM_STATUS_INVALID_PARAMETER;
+        }
+    }
+
+    if (libspdm_get_connection_version(spdm_context) >= SPDM_MESSAGE_VERSION_13) {
+        if ((spdm_context->connection_info.multi_key_conn_rsp) ^ (key_pair_id != 0)) {
```

Comment:
Since this is not really bitwise XORing I'd suggest
```
(spdm_context->connection_info.multi_key_conn_rsp) == (key_pair_id == 0)
```

Reply:
Thanks. I have changed the code to `(spdm_context->connection_info.multi_key_conn_rsp) == (key_pair_id == 0)`.